anyway/parsers/location_extraction.py

```diff
@@ -516,31 +515,44 @@ def extract_location_text(text):
     return text
 
 
-def extract_geo_features(db, newsflash: NewsFlash, update_cbs_location_only: bool) -> None:
-    location_from_db = None
-    if update_cbs_location_only:
-        if newsflash.resolution in ["כביש בינעירוני", "רחוב"]:
-            location_from_db = get_db_matching_location(
-                db, newsflash.lat, newsflash.lon, newsflash.resolution, newsflash.road1
-            )
-    else:
-        newsflash.location = extract_location_text(newsflash.description) or extract_location_text(
-            newsflash.title
+def update_coordinates_and_resolution_using_location_text(newsflash):
+    newsflash.location = extract_location_text(newsflash.description) or extract_location_text(
+        newsflash.title
+    )
+    geo_location = geocode_extract(newsflash.location)
+    if geo_location is not None:
+        newsflash.lat = geo_location["geom"]["lat"]
+        newsflash.lon = geo_location["geom"]["lng"]
+        newsflash.road1 = geo_location["road_no"]
+        newsflash.resolution = set_accident_resolution(geo_location)
+    return geo_location is not None
+
+
+def extract_geo_features(db, newsflash: NewsFlash, use_existing_coordinates_only: bool) -> None:
+    if not use_existing_coordinates_only:
+        update_coordinates_and_resolution_using_location_text(newsflash)
+
+    if newsflash.resolution in [
+        BE_CONST.ResolutionCategories.STREET,
+        BE_CONST.ResolutionCategories.SUBURBAN_ROAD,
```

Comment:
I suggest using SUPPORTED_RESOLUTIONS
It consists of ResolutionCategories.SUBURBAN_JUNCTION, but that's ok since it will be a supported soon)

Reply:
ok, done